cmd/cbuild/commands/build/buildcprj.go

```diff
@@ -43,7 +43,7 @@ func BuildCPRJ(cmd *cobra.Command, args []string) error {
 	debug, _ := cmd.Flags().GetBool("debug")
 	verbose, _ := cmd.Flags().GetBool("verbose")
 	clean, _ := cmd.Flags().GetBool("clean")
-	schema, _ := cmd.Flags().GetBool("schema")
+	schemaChk, _ := cmd.Flags().GetBool("no-schema-check")
```

Comment:
IMHO the negation should be in the variable name:
`noSchemaChk`

Reply:
We use the hyphen as a separator to ensure consistency with other options that already follow this convention, such as `frozen-packs`, `context-set`, and `update-rte`.

Reply:
Addressed in [Renamed variables](https://github.com/Open-CMSIS-Pack/cbuild/pull/336/commits/ff499f2fca4762bab1399b6ff34966c8f7be0f76)